examples/02_decoding/plot_haxby_glm_decoding.py

```diff
@@ -121,11 +121,9 @@
 # and have the :term:`contrast`, we can quickly create a summary report.
 
 from nilearn.image import mean_img
-from nilearn.reporting import make_glm_report
 
 mean_img_ = mean_img(func_filename, copy_header=True)
-report = make_glm_report(
-    glm,
+report = glm.generate_report(
```

Comment:
decided to emphasize more the model.generate_report() method rather than the make_glm_report function to be more consistent with what the maskers do